package.json

```diff
@@ -3,13 +3,15 @@
   "version": "0.1.0",
```

Comment:
This file tells node what the project is and know what dependencies/libraries to load with the project. Right?

Reply:
Yes. When you run "npm install", it installs the versions of software specified in this file to the "node_modules" directory in the project. This is how you can then do something like:
import { BrowserRouter, Route, Switch } from 'react-router-dom'

The react-router-dom has to be included in package.json or that import would fail.